src/js/components/user/presentation/rating.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const Rating = ({ data: averageRating }) => {
+  return <div className='User-rating'>Average rating: {averageRating}</div>
```

Comment:
We should return a <progress> bar: https://css-tricks.com/html5-progress-element/

Since it's a special element it'll have its own block-level className: .ProgressBar